testutil/db.go

```diff
@@ -90,3 +90,17 @@ func tryTestDatabaseLock(ctx context.Context, db *pg.DB) func(context.Context) (
 		return acquired, err
 	}
 }
+
+// TruncateBlockTables ensures the indexing tables are empty
+func TruncateBlockTables(tb testing.TB, db *pg.DB) error {
+	_, err := db.Exec(`TRUNCATE TABLE block_headers`)
+	require.NoError(tb, err, "block_headers")
+
+	_, err = db.Exec(`TRUNCATE TABLE block_parents`)
+	require.NoError(tb, err, "block_parents")
+
+	_, err = db.Exec(`TRUNCATE TABLE drand_block_entries`)
```

Comment:
If the truncation is important to test success, we should require.NoError here too.

Reply:
I think we do on the line below?